# Make the libva depend on libdrm-poulsbo instead of libdrm so that the 
# emenlow libdrm can coexist with the normal libdrm.
# jdike@linux.intel.com

Index: libva-0.31.0/configure.ac
===================================================================
--- libva-0.31.0.orig/configure.ac
+++ libva-0.31.0/configure.ac
@@ -99,14 +99,14 @@ AC_SYS_LARGEFILE
 PKG_CHECK_MODULES([X11], [x11])
 PKG_CHECK_MODULES([XEXT],[xext])
 PKG_CHECK_MODULES([XFIXES], [xfixes])
-PKG_CHECK_MODULES([DRM], [libdrm])
+PKG_CHECK_MODULES([DRM], [libdrm_poulsbo])
 
 PKG_CHECK_MODULES(GEN4ASM, [intel-gen4asm >= 1.0], [gen4asm=yes], [gen4asm=no])
 AM_CONDITIONAL(HAVE_GEN4ASM, test x$gen4asm = xyes)
 
 # Check for libdrm >= 2.4 (needed for i965_drv_video.so)
-if test x$enable_i965_driver = xyes && ! $PKG_CONFIG --atleast-version=2.4 libdrm; then
-    AC_MSG_WARN([libdrm < 2.4 found, disabling build of i965 video driver])
+if test x$enable_i965_driver = xyes && ! $PKG_CONFIG --atleast-version=2.4 libdrm_poulsbo; then
+    AC_MSG_WARN([libdrm_poulsbo < 2.4 found, disabling build of i965 video driver])
     enable_i965_driver=no
 fi
 AM_CONDITIONAL(BUILD_I965_DRIVER, test x$enable_i965_driver = xyes)
